src/test/java/com/google/api/codegen/testdata/discoveries/sheets.v4.json.overrides

```diff
@@ -1,12 +1,267 @@
 {
   "java": {
     "methods": {
+      "sheets.spreadsheets.batchUpdate": {
+        "requestBodyType": {
```

Comment:
Seems like it would be much better if the overrides could just specify names of fields to include, and their properties would be determined from discovery doc. Otherwise API authors writing overrides need to be intimately familiar with our internals to enter all this metadata, some of it redundant even within itself.